test/kubemark/pre-existing/README.md

```diff
@@ -0,0 +1,36 @@
+# Kubemark Pre-existing Provider Guide
+
+## Introduction
+
+The goal of the pre-existing provider is to land kubemark containers onto
+a pre-existing Kubernetes Cluster. It's meant to provide the developer with
+additional flexibility to customize the cluster infrastructure and still use
+the kubemark setup tools.  It is an **advanced** use case and not recommended
```

Comment:
Instead of 'not recommended', maybe 'requires knowledge of how to spin up a k8s master'? :)